x-pack/packages/kbn-elastic-assistant/impl/assistant/assistant_title/index.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FunctionComponent, useMemo, useState } from 'react';
+import {
+  EuiButtonIcon,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiLink,
+  EuiModalHeaderTitle,
+  EuiPopover,
+  EuiText,
+} from '@elastic/eui';
+import { DocLinksStart } from '@kbn/core-doc-links-browser';
+import { FormattedMessage } from '@kbn/i18n-react';
+import * as i18n from '../translations';
+
+export const AssistantTitle: FunctionComponent<{
+  currentTitle: { title: string | JSX.Element; titleIcon: string };
+  docLinks: Omit<DocLinksStart, 'links'>;
+}> = ({ currentTitle, docLinks }) => {
+  const { ELASTIC_WEBSITE_URL, DOC_LINK_VERSION } = docLinks;
+  const url = `${ELASTIC_WEBSITE_URL}guide/en/security/${DOC_LINK_VERSION}/security-assistant.html`;
+
+  const documentationLink = useMemo(
+    () => (
+      <EuiLink
+        aria-label={i18n.TOOLTIP_ARIA_LABEL}
+        data-test-subj="externalDocumentationLink"
+        href={url}
+        rel="noopener"
```

Comment:
If you have `target="_blank"` and `external` you don't `rel="noopener"`/`rel="noreferrer"` as `rel="noopener noreferrer"` will be added automatically.